DashAI/front/src/components/experiments/NewExperimentModal.jsx

```diff
@@ -51,15 +52,47 @@ export default function NewExperimentModal({ open, setOpen }) {
   const [nextEnabled, setNextEnabled] = useState(false);
   const [newExp, setNewExp] = useState(defaultNewExp);
 
+  const uploadRuns = async (experimentId) => {
+    for (const run of newExp.runs) {
+      try {
+        await createRunRequest(
+          experimentId,
+          run.type,
+          run.nickname,
+          run.params,
+          "",
+        );
```

Comment:
I think that this type of operation can be dangerous because it can saturate the server.

Wouldn't it be better for the back to have an endpoint where you send it all the runs in a batch and it would take care of gluing them together?